requirements/requirements-ci.txt

```diff
@@ -123,27 +120,27 @@ pycryptodome==3.8.2
 pyflakes==2.1.1
 pygments==2.4.2
 pyhamcrest==1.9.0
-pyinstaller==3.4
+pyinstaller==3.5
 pylint==2.3.1
 pymacaroons-pynacl==0.9.3
 pynacl==1.3.0
 pyopenssl==19.0.0
-pyparsing==2.4.0
-pyrsistent==0.15.2
-pysaml2==4.7.0
+pyparsing==2.4.1.1
+pyrsistent==0.15.3
+pysaml2==4.8.0
 pysha3==1.0.2
 pytest-forked==1.0.2
 pytest-random==0.2
 pytest-select==0.1.2
-pytest-xdist==1.28.0
-pytest==4.6.3
+pytest-xdist==1.29.0
+pytest==5.0.1
 python-dateutil==2.8.0
 python-magic==0.4.15      # via s3cmd
-pytoml==0.1.20
+pytoml==0.1.21
 pytz==2019.1
 pyyaml==5.1.1
-raiden-contracts==0.26.0
-raiden-webui==0.8.0
+raiden-contracts==0.29.0
+raiden-webui==0.9.1
```

Comment:
Why upgrade the raiden-webui in this PR?

Reply:
I didn't know `requirements/dep upgrade` can take a package name.  Will fix.